### ISSUE TITLE
[hardware::ups::powerware::snmp::plugin] mode "input-lines" does not detect the absence of an input line.

### ISSUE THREAD

**Issue by Aleksey-Maksimov** [centreon/centreon-plugins#4383]
Hello.

A few days ago, at one of the sites, the incoming voltage to the **Eaton Powerware 9125** UPS was lost. 

![изображение](https://user-images.githubusercontent.com/17614847/233990803-7966f32a-b49f-4bc9-9d25-21a8fcf2735d.png)

But the monitoring plug-in in the "input-lines" mode did not show a problem.
I checked and saw that when the input line is missing, the plugin shows the status "OK" and does not show the perfdata.

```
# ./centreon_plugins.pl '--plugin=hardware::ups::powerware::snmp::plugin' '--mode=input-lines' 
'--critical-frequence' '49:51' '--filter-uom' '%' 
'--hostname' '10.1.0.15' '--snmp-community' 'public' '--snmp-version' '1' --verbose

OK: Input Line '1'
```

In a normal situation, when there is an incoming line, the plugin shows the same thing, but only the perfdata appears.

```
# ./centreon_plugins.pl '--plugin=hardware::ups::powerware::snmp::plugin' '--mode=input-lines' 
'--critical-frequence' '49:51' '--filter-uom' '%' 
'--hostname' '10.1.0.15' '--snmp-community' 'public' '--snmp-version' '1' --verbose

OK: frequence: 50.00 Hz - Input Line '1' current: 1.00 A, voltage: 226.00 V | 
'frequence'=50.00;;49:51;; 'current'=1.00;;;0; 'voltage'=226.00;;202:259;;
Input Line '1' current: 1.00 A, voltage: 226.00 V
```

Perhaps there is a problem in the plugin and it does not understand that there is no data on the incoming line?

**Comment by garnier-quentin** [comment]
By default, the mode skips if value is 0. In many cases, 0 means no problem. Maybe i could add an option to display it even if 0.

**Comment by Aleksey-Maksimov** [comment]
_"the mode skips if value is 0"_. I think this is faulty logic.
If the current is 0 hertz, then this does not mean that everything is fine, but that everything is bad, since there is no current.

**Comment by garnier-quentin** [comment]
Yes. But on some UPS, the value is 0 if OID is not managed by the equipment.

**Comment by Aleksey-Maksimov** [comment]
But now the plugin is not working as expected. We have the --critical-frequence' '49:51' option, which means that frequence can be between 49 and 51. 0 is not in this range and cannot be a sign of a normal state.

**Comment by Aleksey-Maksimov** [comment]
Hello Quentin.
How can we solve this problem?

**Comment by Aleksey-Maksimov** [comment]
snmpwalk will not give you anything if the UPS is running from the input line. The problem occurs when the incoming line is disconnected. All this is described above.

**Comment by lucie-tirand** [comment]
Hello :)

This issue sould be solved with the [latest release 20260600](https://github.com/centreon/centreon-plugins/releases/tag/plugins-20260600).
